### PR TITLE
Setup automatic PyPI releases via Git tags.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

File: src/jupyter_spark/__init__.py [hunk 1]
```diff
@@ -21,6 +26,9 @@ def _jupyter_server_extension_paths():  # pragma: no cover
 
 
 def load_jupyter_server_extension(nbapp):  # pragma: no cover
+    from .spark import Spark
+    from .handlers import SparkHandler
```
[the remark this answers is not in the document]
Comment (mdboom) [reply src/jupyter_spark/__init__.py]: Good point.

**Comment by mdboom** [comment]
This looks good.  I don't know if there's any way to test it short of merging and then making a tag...

**Comment by jezdez** [comment]
@mdboom This is along the lines of many automatic release setups I've done, except for the use of the new build stages. So let's hope for the best :)

Do you want to do a release then?

**Comment by mdboom** [comment]
Sure, I'll tag a release, maybe after #37 is merged?

**Pull request merged**